w09h01/test/pgdp/pingutrip/LegalCheck.java

```diff
@@ -0,0 +1,239 @@
+import java.util.ArrayList;
```

Comment:
```suggestion
package pgdp.pingutrip;

import java.util.ArrayList;
```

Package is missing, right?

Reply:
not anymore